example/cairo/src/lib.cairo

```diff
@@ -1,7 +1,7 @@
 mod utils;
 
-fn main(n: felt252) -> Array<felt252> {
-    array![utils::factorial(n)]
+fn main(n: felt252, c: felt252) -> Array<felt252> {
```

Comment:
why `c`?

Reply:
and the name `c` is from analogy to the integral

Reply:
g!